kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -207,14 +208,34 @@ server {
 
             os.rename("/srv/storage/" .. tmp_file_name, "/srv/storage/" .. header_path_with_filename)
             os.remove("/srv/storage/" .. tmp_file_name .. ".shactx")
+
+            -- Send confirmation to the conductor after uploaded file.
+            local http = require "resty.http"
+            local httpc = http.new()
+            local res, err = httpc:request_uri("{{ upload_report_url }}" .. header_path_with_filename, {
+            method = "POST",
+            })
```

Comment:
What happens if the connection fails? For example if network is down or something is blocking the connection?

Reply:
I forgot about it. I will add a condition.
```
            if not res then
                ngx.status = 500
                ngx.say('{"error_message": "failed to request:' .. err .. '", "error_code": "failed request"}')
                return ngx.exit(500)
            end

```

Reply:
Is `res` a boolean value? If not, then this condition should be more specific.

Also it should be more like `"HTTP request to conductor failed: "`

Reply:
According to https://github.com/pintsized/lua-resty-http#request, return table or nil. Condition above is from documentation example: https://github.com/pintsized/lua-resty-http#synopsis

Reply:
Changed `error_message`

Reply:
It's the same story as with `if x` vs `if x is not None` in Python. Some people like it because it's shorter to type but our convention is to check conditions explicitly - there are corner cases (at least in Python) that can easily trip you up if you're not careful.

So this should be something like `if res != nil`.

Reply:
Ok.  I will change it, but I'm not sure if there is the same analogy in lua. This condition `if not res then` comes from the documentation, so it must work.

Reply:
I just realized that the condition should look like this: `if res == zero` or old version `if not res`. Which one we want to used ?

Reply:
This `if not res` condition is correct when `res` is nil, so we can choose between two options that I mentioned above.